packages/calypso-build/babel.config.js

```diff
@@ -29,16 +29,6 @@ const config = {
 	env: {
 		build_pot: {
 			plugins: [
-				[
-					'@wordpress/babel-plugin-makepot',
-					{
-						output: 'build/i18n-calypso/gutenberg-strings.pot',
-						headers: {
-							'content-type': 'text/plain; charset=UTF-8',
-							'x-generator': 'calypso',
-						},
-					},
-				],
 				[
 					'@automattic/babel-plugin-i18n-calypso',
```

Comment:
I should move this one to Calypso root since I don't think we expect/encourage use of `i18n-calypso` in SDK built blocks and the like.